packages/workflow/src/workflow.ts

```diff
@@ -112,11 +113,14 @@ function timerNextHandler(input: TimerInput) {
     }
     activator.pushCommand({
       startTimer: {
-        seq: input.seq,
-        startToFireTimeout: msToTs(input.durationMs),
+        seq,
+        startToFireTimeout: msToTs(durationMs),
+      },
+      userMetadata: options && {
```

Comment:
```suggestion
      userMetadata: options?.summary && {
```

Then you can remove the inner terniary